cnd/migrations/2020-04-09-030124_create-message-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE dial_infos
+(
+        id             INTEGER NOT NULL PRIMARY KEY,
+        local_swap_id  UNIQUE NOT NULL,
+        peer_id        NOT NULL
+);
+
+CREATE TABLE han_ethereum_created_swaps
```

Comment:
We need to somehow track, whether the information in these tables is alpha or beta ledger.

We could:

- have a field in each of these tables that says alpha/beta
- have a field in the `swaps` table proposed above that says the protocol name for alpha/beta
- more options?